src/cmds/cloud/setData.js

```diff
@@ -66,7 +54,12 @@ yargs
           },
         });
     },
-    handler: (args) => {
-      setData(args);
+    handler: async (args) => {
+      try {
+        await setData(args);
```

Comment:
Same as `listThings`, I'm not understanding if this is being logged or returned anywhere.... anyway, I believe we should always log successful operations too!